### DISCUSSION TITLE
Does Harlem support cross-tab synchronization?

### DISCUSSION THREAD

**Discussion by Lexpeartha** [andrewcourtice/harlem#49] [Q&A] [answered]
More specifically, I'm wondering whether or not does Harlem (or it's storage extension) support storing in storage across multiple tabs.
This library seems pretty interesting, and I'm considering using it in my project, however cross-tab syncing is required for my use case.

Also, awesome work with the library so far, I really like it's unopinionated nature 🙌🏽

**Comment by andrewcourtice** [comment, marked as answer]
Hi @Lexpeartha 

Thank you for your kind words and support.

Yes, the storage extension supports tab synchronisation 😄 You can enable/disable this feature by setting the `sync` value in the storage extension's options. By default `sync` is enabled so you won't have to do anything to turn it on. See the [extension docs here](https://harlemjs.com/extensibility/extensions/storage.html#options) for more info about the options available for the storage extension.

If you would like to see a demo of the tab sync in action, open the [Harlem demo app](https://andrewcourtice.github.io/harlem/) in 2 different tabs and change some options.

Here is a demo of it in action:
https://user-images.githubusercontent.com/11718453/155244281-55e9cfe2-a884-4bf8-afaf-7aba653326d7.mp4

Thanks 😄
